### PR TITLE
Remove rhel_alts script and update facts for OracleLinux 8 and 9

### PR DESCRIPTION
**Pull request by yakatz**

### CODE REVIEW

File: facts/Vagrantfile [hunk 1]
```diff
@@ -177,6 +177,22 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision 'shell', path: 'get_facts.sh'
     host.vm.provision 'shell', inline: '/sbin/shutdown -h now'
   end
+  config.vm.define 'oraclelinux-8-x86_64', autostart: false do |host|
+    host.vm.box = 'bento/oraclelinux-8'
+    host.vm.synced_folder '.', '/vagrant'
+    host.vm.provision 'file', source: 'Gemfile', destination: 'Gemfile'
+    host.vm.provision 'shell', inline: 'sysctl -w net.ipv6.conf.all.disable_ipv6=0'
```
Comment (ekohl) [inline facts/Vagrantfile:184]: https://github.com/voxpupuli/facterdb/pull/340 was merged that makes this line redundant.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: facts/Vagrantfile [hunk 2]
```diff
@@ -177,6 +177,22 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision 'shell', path: 'get_facts.sh'
     host.vm.provision 'shell', inline: '/sbin/shutdown -h now'
   end
+  config.vm.define 'oraclelinux-8-x86_64', autostart: false do |host|
+    host.vm.box = 'bento/oraclelinux-8'
```
[the remark this answers is not in the document]
Comment (yakatz) [reply facts/Vagrantfile]: The Oracle boxes aren't in Vagrant Cloud. I found them on Oracle's site.

[the remark this answers is not in the document]
Comment (bastelfreak) [reply facts/Vagrantfile]: have you tested them?

```
host.vm.box = "oraclelinux/8"
host.vm.box_url = "https://oracle.github.io/vagrant-projects/boxes/oraclelinux/8.json"

[the remark this answers is not in the document]
Comment (yakatz) [reply facts/Vagrantfile]: Running right now, I will let you know

[the remark this answers is not in the document]
Comment (yakatz) [reply facts/Vagrantfile]: Testing is done, results are exactly the same between the bento and official boxes

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**